internal/api/deploymentscheduler_test.go

```
@@ -0,0 +1,918 @@
+		for _, m := range sm.Metrics {
+			if m.Name != api.MetricDeploymentFires {
+				continue
+			}
+			s, ok := m.Data.(metricdata.Sum[int64])
+			if !ok {
+				t.Fatalf("%s is %T, want an int64 sum", api.MetricDeploymentFires, m.Data)
+			}
+			for _, dp := range s.DataPoints {
+				got := map[string]string{}
+				for _, kv := range dp.Attributes.ToSlice() {
+					got[string(kv.Key)] = kv.Value.Emit()
```

golangci-lint (2.12.2)

[error] SA1019: kv.Value.Emit is deprecated: Use [Value.String] instead. 

(staticcheck)